frontend/public/components/modals/create-namespace-modal.jsx

```diff
@@ -98,7 +98,7 @@ const CreateNamespaceModal = connect(null, mapDispatchToProps)(class CreateNames
   render() {
     const label = this.props.createProject ? 'Project' : 'Namespace';
     const defaultNetworkPolicies = {
-      [allow]: 'No restrictions (default)',
+      [allow]: 'No restrictions',
```

Comment:
I think we're losing some meaning here. "default" doesn't mean this is the default option. It means this is the behavior when there is no network policy. 

cc @serenamarie125

Reply:
So would it be more accurate for the selection to say "None - No restrictions" 

current screenshot for reference 

---

<img width="601" alt="Screen Shot 2019-08-08 at 2 41 07 PM" src="https://user-images.githubusercontent.com/1874151/62728808-ae580280-b9ea-11e9-9355-85f780b5b782.png">

Reply:
It's probably OK as in the PR.